hw/ip/aes/doc/_index.md

```diff
@@ -313,16 +313,20 @@ This section discusses how software can interface with the AES unit.
 ## Initialization
 
 Before initialization, software must ensure that the AES unit is idle by checking {{< regref "STATUS.IDLE" >}}.
-If the AES unit is not idle, write operations to {{< regref "CTRL" >}}, the Initial Key registers {{< regref "KEY0" >}} - {{< regref "KEY7" >}} and initialization vector (IV) registers {{< regref "IV0" >}} - {{< regref "IV3" >}} are ignored.
+If the AES unit is not idle, write operations to {{< regref "CTRL" >}}, the Initial Key registers {{< regref "KEY_SHARE00" >}} - {{< regref "KEY_SHARE17" >}} and initialization vector (IV) registers {{< regref "IV0" >}} - {{< regref "IV3" >}} are ignored.
```

Comment:
Is it "weird" for software that there is a "hole" between KEY_SHARE07 and KEY_SHARE10 ?  Should we make the names a little clearer (either SHAREA/SHAREB instead of SHARE0/SHARE1, or just adding some _ to the names)?  Or is this not needed? (I'm honestly not sure here.)

Reply:
Thanks for raising this @cdgori . I agree with you that KEY_SHARE0_7 and KEY_SHARE1_7 would be better. I prefer the `_` instead of Share A and Share B. I have created another PR to modify the regtool for this here #2980 but it seems that needs to some more work.